test/utils-test.ts

```diff
@@ -1,39 +0,0 @@
-/**
- * Copyright 2017 Google Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-'use strict';
-
-import * as assert from 'assert';
-import {toWinDirFormat, getLocalAppDataPath} from '../src/utils';
-
-describe('WSL path format to Windows', () => {
```

Comment:
maybe we can at least leave a WSL test that this works and just skip it in non-wsl environments? documenting what it's supposed to do is still valuable even if it can't be run in CI atm

Reply:
For sure - I'm thinking I'll try and do some mocking, because we should be able to test the path separators that are used in the respective envs.

If you can mock/force whatever check that reports what OS you're on, you can write at least some basic tests :)